sunpycube/cube/NDCube.py

```diff
@@ -0,0 +1,238 @@
+from sunpy.visualization.imageanimator import ImageAnimatorWCS
+from collections import OrderedDict, namedtuple
+import matplotlib.pyplot as plt
+import sunpy.visualization.wcsaxes_compat as wcsaxes_compat
+import astropy.units as u
+import sunpycube.cube.cube_utils
+import sunpycube.wcs_util
+import astropy.nddata
+import numpy as np
+import copy
+
+PixelPair = namedtuple('PixelPair', 'dimensions axes')
```

Comment:
This is a neat object!  But can we change this line to ```DimensionPair = namedtuple('DimensionPair', 'shape axes_types')```?  And of course, make the corresponding changes in NDCube.dimensions.

Great stuff!